src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -97,6 +97,20 @@ def load_arguments(self, _):
     ui_form_definition_file_type = CLIArgumentType(options_list=['--ui-form-definition'], completer=FilesCompleter(), type=file_type,
                                                    help="A path to a uiFormDefinition file in the file system")
 
+    stacks_name_type = CLIArgumentType(options_list=['--name', '-n'], help='The name of the deployment stack.')
+    stacks_description_type = CLIArgumentType(options_list=['--description'], help='The description of deployment stack.')
+    stacks_stack_type = CLIArgumentType(help='The deployment stack resource id.')
+    stacks_stack_name_type = CLIArgumentType(help='The deployment stack name')
+    stacks_stack_deployment_resource_group = CLIArgumentType(options_list=['--deployment-resource-group', '--dr'], help='The scope at which the initial deployment should be created. If a scope is not specified, it will default to the scope of the deployment stack.')
+    stacks_stack_deployment_subscription = CLIArgumentType(options_list=['--deployment-subscription', '--ds'], help='The scope at which the initial deployment should be created. If a scope is not specified, it will default to the scope of the deployment stack.')
+    stacks_delete_resources_type = CLIArgumentType(arg_type=get_three_state_flag(), options_list=['--delete-resources'], help='Flag to indicate delete rather than detach for the resources.')
+    stacks_delete_resource_groups_type = CLIArgumentType(arg_type=get_three_state_flag(), options_list=['--delete-resource-groups'], help='Flag to indicate delete rather than detach for the resource groups.')
+    stacks_delete_all_type = CLIArgumentType(arg_type=get_three_state_flag(), options_list=['--delete-all'], help='Flag to indicate delete rather than detach for the resources and resource groups.')
+    stacks_deny_settings_mode = CLIArgumentType(arg_type=get_enum_type(['none', 'denyDelete', 'denyWriteAndDelete']), help='Define which operations are denied on resources managed by the stack: denyDelete or denyWriteAndDelete.')
```

Comment:
After defining the `get_enum_type` type, it is not necessary to specify the available values of the parameter.
```suggestion
    stacks_deny_settings_mode = CLIArgumentType(arg_type=get_enum_type(['none', 'denyDelete', 'denyWriteAndDelete']), help='Define which operations are denied on resources managed by the stack.')
```